modules/react-native-google-analytics/README.md

```diff
@@ -45,440 +91,86 @@ dependencies {
-  });
-```
-#### 41. setUserProperty
-Sets a key/value pair of data on the current user. Each Firebase project can have up to 25 uniquely named (case-sensitive) user properties.
-```
-setUserProperty(name: string, value: string);
-```
-
-#### 42. getAppInstanceId
-Retrieve the app instance id of the application.
-```
-getAppInstanceId();
-```
-
-### Reserved Events
-Once you have integrated Google Analytics Firebase SDK in your app, a number of events are automatically reported to Firebase. These event names are called as 'Reserved Events'.
-
-1. ad_activeview
-2. ad_click
-3. ad_impression
-4. d_query
-5. adunit_exposure	
-6. app_background
-7. app_clear_data
-8. app_remove	
-9. app_store_refund	
-10. app_store_subscription_cancel
-11. app_store_subscription_convert	
-12. app_store_subscription_renew	
-13. app_update
-14. app_upgrade
-15. dynamic_link_app_update
-16. dynamic_link_first_open
-17. error
-18. dynamic_link_app_open
-19. first_open
-20. first_visit
-21.	in_app_purchase
-22.	notification_dismiss
-23. notification_foreground
-24. notification_open
-25. notification_receive
-26. ad_exposure
-27. os_update
-28.	session_start
-29.	session_start_with_rollout
-30. user_engagement
-31. ad_reward	
-
-### Disable Auto-Initialization
-1. Analytics can be further configured to disable auto collection of Analytics data. This is possible by setting the below noted property on the firebase.json file at the root of your project directory.
-
-project-root/firebase.json
-```
-{
-  "react-native": {
-    "analytics_auto_collection_enabled": false
-  }
-}
-```
-2. To re-enable analytics (e.g. once you have the users consent), call the setAnalyticsCollectionEnabled method:
-```
-setAnalyticsCollectionEnabled(true)
-```
-### Disable screenview tracking
-Automatic screenview reporting can be turned off/on through google_analytics_automatic_screen_reporting_enabled property of firebase.json file.
-```
-{
-  "react-native": {
-    "google_analytics_automatic_screen_reporting_enabled": false
-  }
-}
-```
-
-
-https://user-images.githubusercontent.com/76822297/176847557-ba7c3b34-5213-4cf0-9090-930fb2919122.mp4
-
-
+### iOS setup
+
+On the Firebase console, add a new iOS application and enter your projects details. The "iOS bundle ID" must match your local project bundle ID. The bundle ID can be found within the "General" tab when opening the project with Xcode.
+Download the `GoogleService-Info.plist` file.
+
+1. Using Xcode, open the projects /ios/{projectName}.xcodeproj file (or /ios/{projectName}.xcworkspace if using Pods).
+
+   Right click on the project name and "Add files" to the project, as demonstrated below:
+   ![image](https://github.com/cbshoaib/modules/assets/76822297/d9e5db27-4279-4c96-bbe8-c27b79691e68)
+
+Select the downloaded GoogleService-Info.plist file from your computer, and ensure the "Copy items if needed" checkbox is enabled.
+![image](https://github.com/cbshoaib/modules/assets/76822297/8f9c8db2-9d3a-4b51-825d-8f36bc112099)
```

Comment:
Could you please upload those to this repo?

Reply:
These images are also present in the official documentation of react-native-firebase. I have replaced the image links with the ones that are being used in that documentation.

[documentation](https://rnfirebase.io/)